packages/create-svelte/test/check.js

```diff
@@ -47,14 +47,56 @@ const workspace = {
 	pnpm: { overrides },
 	devDependencies: overrides
 };
+
 fs.writeFileSync(
 	path.join(test_workspace_dir, 'package.json'),
 	JSON.stringify(workspace, null, '\t')
 );
+
 fs.writeFileSync(path.join(test_workspace_dir, 'pnpm-workspace.yaml'), 'packages:\n  - ./*\n');
 
-for (const template of fs.readdirSync('templates')) {
-	if (template[0] === '.') continue;
+const exec_async = promisify(exec);
+
+beforeAll(
+	() =>
+		exec_async(`pnpm install --no-frozen-lockfile`, {
+			cwd: test_workspace_dir
+		}),
+	60000
+);
+
+function patch_package_json(pkg) {
```

Comment:
but they _are_ global, or rather shared by everything in the module. not sure what this would achieve — `overrides` would still be defined/populated at the module level, we'd just be replacing a closed-over value with an identical parameter, no?

Reply:
yes. I just think it's a bit cleaner to avoid global values. no functional difference though and just a matter of opinion